easybuild/easyconfigs/p/PyWavelets/PyWavelets-1.8.0-foss-2024a.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name = 'PyWavelets'
+version = '1.8.0'
+
+homepage = 'https://pywavelets.readthedocs.io'
+description = "PyWavelets is open source wavelet transform software for Python."
+
+toolchain = {'name': 'foss', 'version': '2024a'}
+
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['f3800245754840adc143cbc29534a1b8fc4b8cff6e9d403326bd52b7bb5c35aa']
+
+dependencies = [
+    ('Python', '3.12.3'),
+    ('SciPy-bundle', '2024.05'),  # for numpy
+    ('meson-python', '0.16.0'),
+    ('Cython', '3.0.10'),
```

Comment:
this should be a build dependency instead.

Reply:
Thanks! Fixed.